charts/velero/templates/deployment.yaml

```diff
@@ -99,6 +99,9 @@ spec:
             {{- if .defaultVolumesToRestic }}
             - --default-volumes-to-restic
             {{- end }}
+            {{- with .defaultResticMaintenanceFrequency }}
```

Comment:
Can we change the name to align with the flag? `defaultResticPruneFrequency`

Reply:
Sure